packages/salesforcedx-vscode-core/src/commands/functions/functionService.ts

```diff
@@ -12,6 +12,15 @@ import { Disposable } from 'vscode';
 import { workspaceContext } from '../../context';
 import { getRootWorkspace, getRootWorkspacePath } from '../../util';
 
+/**
+ * An enum for the different types of functions.
+ */
+export enum FUNCTION_TYPE {
```

Comment:
I think the convention for enums in TS is PascalCase, so:
```
export enum FunctionType {
  JavaScript,
  TypeScript,
  Java
}
```

Here's what the TypeScript docs say: https://www.typescriptlang.org/docs/handbook/enums.html